com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/internal/ash/AshFrameHandler.java

```diff
@@ -163,6 +164,7 @@ public void run() {
 
                 int exceptionCnt = 0;
 
+                boolean rejectionCondition = false;
```

Comment:
Can I suggest to just add a comment here - something like.

```
// Ensure that a NAK is only sent for the first error in a sequence and a valid response is required before sending another NAK
```

Just to remind us why this is here :)

Reply:
done